hips/tiles/tests/test_surveys.py

```diff
@@ -0,0 +1,22 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
+from astropy.tests.helper import remote_data
+from ..surveys import HipsSurveyPropertyList
+
+class TestHipsSurveyPropertyList:
+    @classmethod
+    def setup_class(cls):
+        filename = get_pkg_data_filename('data/surveys.txt')
+        cls.hips_survey_list = HipsSurveyPropertyList.read(filename)
+
+    @remote_data
+    def test_fetch(self):
+        self.hips_survey_list.fetch()
+
+    def test_surveys(self):
+        len_precomp = 4
```

Comment:
Please don't use the "precomp" name, it's not very common. Common generic names are "expected" or "desired". In this case though, I think don't need the local variable (just used once, on a line that's already short, just copy the value here).